client/src/App.js

```diff
@@ -17,7 +18,8 @@ function App() {
         <Route exact path="/" component={LandingPage} />
         <Route path="/login" component={LoginPage} />
         <Route path="/signup" component={SignUpPage} />
-        <Route path="/profile" component={ProfilePage} />
+        <Route path="/profile" component={EditProfilePage} />
+        <Route path="/profile-payment" component={ProfilePayment} />
```

Comment:
You should change this route path and associated routes to just "payment", to keep a consistent routing style